src/nautilus-server/src/main.rs

```diff
@@ -18,11 +18,20 @@ async fn main() -> Result<()> {
     // This value can be stored with secret-manager. To do that, follow the prompt `sh configure_enclave.sh`
     // Answer `y` to `Do you want to use a secret?` and finish.
     // Then uncomment this code instead to fetch from env var API_KEY, which is fetched from secret manager.
-    let api_key = std::env::var("API_KEY").expect("API_KEY must be set");
-    // let api_key = "045a27812dbe456392913223221306".to_string();
+    // let api_key = std::env::var("API_KEY").expect("API_KEY must be set");
```

Comment:
needs an update?

Reply:
yes this should be behind the feature flag, revised